src/mpid/ch3/channels/nemesis/netmod/ofi/ofi_init.c

```diff
@@ -335,6 +335,7 @@ static inline int compile_time_checking()
     MPL_COMPILE_TIME_ASSERT(sizeof(MPID_nem_ofi_vc_t) <= MPIDI_NEM_VC_NETMOD_AREA_LEN);
     MPL_COMPILE_TIME_ASSERT(sizeof(MPID_nem_ofi_req_t) <= MPIDI_NEM_REQ_NETMOD_AREA_LEN);
     MPL_COMPILE_TIME_ASSERT(sizeof(iovec_t) == sizeof(MPL_IOV));
+    MPL_COMPILE_TIME_ASSERT(MPIR_CONTEXT_ID_BITS <= 16);
```

Comment:
Add a comment such as "ch3 does not support extended context id bits"?

Reply:
Will do.